app/controllers/devise/devise_authy_controller.rb

```diff
@@ -121,25 +121,17 @@ def GET_verify_authy_installation
   end
 
   def POST_verify_authy_installation
-    token = Authy::API.verify({
-      :id => self.resource.authy_id,
-      :token => params[:token],
-      :force => true
-    })
+    token_valid = registration_token_valid?(resource.mfa_config)
 
-    self.resource.authy_enabled = token.ok?
+    resource.authy_enabled = token_valid
 
-    if token.ok? && self.resource.save
-      remember_device(@resource.id) if params[:remember_device].to_i == 1
+    if token_valid && resource.save
+      remember_device(resource.id) if params[:remember_device].to_i == 1
       record_authy_authentication
```

Comment:
need to rename this away from authy since it's verify now

Reply:
Done with this one.